src/v3/src/components/PhoneAuthenticator/PhoneAuthenticator.tsx

```diff
@@ -62,26 +66,38 @@ const PhoneAuthenticator: UISchemaElementComponent<{
     return `${code}${phone}`;
   };
 
+  const handleBlur = () => {
+    setIsTouched(true);
+    onValidateHandler(setFieldError, phone);
+  };
+
   useEffect(() => {
     onChangeHandler(formatPhone(phone, phoneCode, extension));
+    onValidateHandler(setFieldError, phone);
```

Comment:
do we need phone validation for v2 parity? I remember we decided to not validate phone number client side due to there is no concrete way to handle formats in all regions

Reply:
Yes, because this doesnt validate whether the phone is correct or not, it just validates the existence of a value. 

Additionally, if they click submit without a value, currently, the error continues to display in the field until they submit again. This is not parity with v2. The error should be removed as soon as they start typing into the field (resolving the error). This validation hook handles that. 

See: https://oktainc.atlassian.net/browse/OKTA-497119

Reply:
hmm, this does not seems to belong to the validation logic, and handle it in a generic hook may cause other validation logic leaks to this component. maybe some logic to just handle clear message? also, possible to handle it in a separate PR?

Reply:
Current Widget doesn't just clear message on field, it re-displays it if the field is cleared. I am not sure we should remove functionality that the current widget has. 

This change is related to this ticket: https://oktainc.atlassian.net/browse/OKTA-488781

Reply:
If we need this for parity for beta let's get this PR in soon so I'd lean towards not separating the PRs at this point in the interest of time, since this PR is already well-reviewed. I think in the future we can keep getting better at breaking work apart but for this instance we should proceed in here.

As far as the behavior/hook code, again, let's get functionality working for now since I don't see any major downside to getting this in now and then reworking down the line as we do more validation work if necessary.